corehq/apps/enterprise/enterprise.py

```diff
@@ -422,7 +422,9 @@ def __init__(self, account, couch_user, start_date=None, end_date=None, num_days
     def total_for_domain(self, domain_obj):
         query = SMS.objects.filter(
             domain=domain_obj.name,
+            processed=True,
```

Comment:
What does the `processed` attribute mean in SMS class?

Reply:
I believe it gets set when a message completes processing -- you can see that [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/tasks.py#L111-L119). One example of why we need this would be code like [this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/tasks.py#L281-L289), where the message doesn't have an `error` yet, but we are in the process of retrying it. That message shouldn't be reported yet, so we filter it out by requiring `processed` to be `True`